core/lib/state/Cargo.toml

```diff
@@ -22,6 +22,8 @@ tokio = { version = "1", features = ["rt"] }
 tracing = "0.1"
 itertools = "0.10.3"
 
+metrics = "0.21"
```

Comment:
I believe we have migrated from the `metrics` crate, and moreover the changes in this PR do not use it.